web/src/server_events.js

```diff
@@ -264,7 +265,14 @@ export function force_get_events() {
 export function home_view_loaded() {
     waiting_on_homeview_load = false;
     get_events_success([]);
-    $(document).trigger("home_view_loaded.zulip");
```

Comment:
This seems like an old event trigger that no longer does anything. A git-grep search for `home_view_loaded.zulip` only returns this.

Reply:
`git log -Shome_view_loaded.zulip` would be a good way to check this.

Reply:
That was very helpful! Seems like the event code was removed in commit 92ad98814429c8854c725278c55bf976b927ad5c.

Reply:
This was taken care of in #26122.